src/components/PurchaseOrder/util.js

```diff
@@ -70,3 +70,11 @@ export const getAddresses = (addresses) => {
     };
   });
 };
+
+export const getVendors = (vendors) => {
+  return vendors.filter(vendor => !vendor.isVendor && vendor.status === 'Active')
+    .map(vendor => ({
+      value: vendor.id,
+      label: vendor.name,
```

Comment:
I think label should be more like this https://github.com/folio-org/ui-invoice/blob/master/src/common/utils/getOptions.js#L3